nmaipy/feature_api.py

```diff
@@ -144,6 +144,7 @@ def __init__(
         alpha: Optional[bool] = False,
         beta: Optional[bool] = False,
         prerelease: Optional[bool] = False,
+        only3d: Optional[bool] = False,
```

Comment:
The docstring of the args needs to be updated for this function

Reply:
Sorted